src/main/cc/ConfEcalc/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ SET(CMAKE_CXX_FLAGS -pedantic-errors)
 # get paths into the parent Osprey project
 set(OspreyPath ${CMAKE_SOURCE_DIR}/../../../..)
 set(JnaPlatform linux-x86-64) # TODO: get the JNA names for each platform
+#set(JnaPlatform darwin-aarch64) #for apple silicon m1 chips
```

Comment:
Ideally, we'd have some kind of logic here that picks the correct value for the current platform automatically. But I don't even know how to do that (I'm not much of a CMake expert), so probably leaving this comment here is fine for now.